src/components/Renderer/RendererPlayground/index.tsx

```diff
@@ -129,7 +128,7 @@ const RendererPlaygroundBase = ({
 	}, [uiSchemaStr]);
 
 	React.useEffect(() => {
-		const example = get(examples, selectedExample, defaultJsonProps);
+		const example = examples?.[selectedExample] ?? defaultJsonProps;
```

Comment:
If this was an explicit null before then the `example.value` below would have thrown a TypeError so I'm pretty confident the `??` is safe :P